kubernetes/cray-opa/tests/opa/spire_test.rego.tpl

```diff
@@ -69,10 +82,6 @@ spire_correct_ncn_sub(sub) {
 
   # Validate that only CFS can access
   allow.http_status == 403 with input as {"attributes": {"request": {"http": {"method": "PATCH", "path": cfs_ncn_mock_path, "headers": {"authorization": sub}}}}}
-
-  # Validate that DVS can access SoftwareStatus
-  # not allow.http_status with input as {"attributes": {"request": {"http": {"method": "PATCH", "path": "/apis/smd/hsm/v2/State/Components/ncnw001/SoftwareStatus", "headers": {"authorization": sub}}}}}
```

Comment:
Why remove "PATCH"?

Reply:
I didn't remove it, I just moved it up in the code so that the smd/hsm apis were closer together for better maintenance.